packages/eslint-config-angular/template.js

```diff
@@ -86,5 +94,18 @@ module.exports = {
             Providing a `trackBy` function in `ngFor` loops can improve performance in specific cases where Angular can't track references, but it's overkill to require it for every `ngFor` so this rule is disabled.
         */
         '@angular-eslint/template/use-track-by-function': 'off'
-    }
+    },
+    overrides: [
```

Comment:
Yes I think a post mentioning this change, mentioning what changes to apps are recommended, and reminding authors and reviewers to be observant for existing i18n tags on attributes that should not be localized, because this change does nothing to flag existing or new instances of that. 

I'll plan to make that post.

Reply:
sounds good to me, thanks @rbell517 👍